VCFv4.3.tex

```diff
@@ -503,7 +506,41 @@ \subsubsection{Genotype fields}
   All phased genotypes that do not contain a PS subfield are assumed to belong to the same phased set.
   If the genotype in the GT field is unphased, the corresponding PS field is ignored.
   The recommended convention is to use the position of the first variant in the set as the PS identifier (although this is not required).
-\end{itemize}
+  \item LAA is a sorted list of n distinct integers, where 1 <= n <= $|ALT|$, giving the (1-based) indices within ALT of the alleles that are observed in the sample. 
```

Comment:
**1 <= n <= $|ALT|$** converts to **1 ¡= n ¡= |ALT |** in the PDF.  Is this what was intended? (Not a notation I have seen before if so).

Reply:
PR #745 corrects the formatting of the mathematics.